extension/cart/transformToShopgateCart.js

```diff
@@ -201,6 +201,17 @@ function getCartItems (magentoCart, shopgateProducts) {
         }
       }
 
+      if (magentoCart.items[i].options) {
+        magentoCart.items[i].options.filter(option => {
+          const allowedTypes = ['text', 'area']
```

Comment:
why are these the only ones allowed? Is it because we do not support any other in the frontend?

Reply:
No, currently we only support text like "text" and "area"